manifests/init.pp

```diff
@@ -312,6 +312,7 @@
     'x86_64', 'amd64': { $real_arch = 'amd64' }
     'i386':            { $real_arch = '386' }
     'aarch64':         { $real_arch = 'arm64' }
+    'arm64':           { $real_arch = 'arm64' }
```

Comment:
The `node_exporter` installed for darwin arm64 (node_exporter-1.6.1.darwin-arm64.tar.gz) is not working, bad architecture. So better use amd64 instead to run in Apples Rosetta mode.